specification/source/cdl.rst

```diff
@@ -1688,118 +1688,97 @@ Therefore, tools that process CDL can infer the following information:
 
 .. _sec_tag_pro:
 
-Tagged Properties
+Semantic Information
 .................
 
-The buildings industry uses different tagging schemes such as
-Brick (https://brickschema.org/) and Haystack (https://project-haystack.org/).
-CDL allows, but does not require, use of the Brick or Haystack tagging scheme.
+The buildings industry uses different metadata schemes such as
+`Brick <https://brickschema.org/>`_, `Project Haystack <https://project-haystack.org/>`_ and `ASHRAE 223p <https://www.ashrae.org/about/news/2018/ashrae-s-bacnet-committee-project-haystack-and-brick-schema-collaborating-to-provide-unified-data-semantic-modeling-solution>`_. CDL allows, but does not require, use of the Brick or Haystack or any other tagging scheme.
 
-CDL allows to add tags to declarations that instantiate
-
-* elementary blocks (:numref:`sec_ele_blo`), and
-* composite blocks (:numref:`sec_com_blo`).
-
-[We currently do not see a use case that would require
-adding a tag to a ``parameter`` declaration.]
+* Semantic information will be included within the ``annotation`` keyword, using the ``__Buildings`` or ``__cdl`` annotation. ``__cdl`` will be used when the semantic information is regarding a control sequence (inputs, outputs etc.) and ``__Buildings`` will be used for every other instance that are not a part of the control sequence (equipment, zones etc.). The following instances defined in CDL can have annotations containing semantic information:
+ * Inputs
+ * Outputs
+ * Parameters
+ * Composite blocks (:numref:`sec_com_blo`)
+ * Packages
+ 
+* All semantic information will be included under the ``semantic`` section within the ``__Building`` and/or ``__cdl`` annotations. 
```

Comment:
I don't understand why line 1704 and below are also bullets. Is that intentional and if so, what is the paragraph structure?

Reply:
I kept bullets+paragraphs because it looked more organized, but I've changed it to only paragraphs.